src/theme/index.tsx

```diff
@@ -36,6 +36,7 @@ const BREAKPOINTS = {
   xxxl: 1920,
 }
 
+// deprecated - please use the animations.ts file
```

Comment:
Deprecate this with jsdoc (`/** @deprecated */`; https://jsdoc.app/tags-deprecated.html) so that it will be more apparent in modern editors.